flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -334,7 +334,9 @@ public void onUnknownDeploymentsOf(
                         .createSlotPoolService(
                                 jid,
                                 createDeclarativeSlotPoolFactory(
-                                        jobMasterConfiguration.getConfiguration()));
+                                        jobMasterConfiguration.getConfiguration()),
+                                null,
```

Comment:
Why not set it according to the configuration?

Reply:
hi, @KarmaGYZ Thank you very much for your comment.

Did you mean that we didn't decide to pass the values of `slotRequestMaxInterval`  `slotBatchAllocatable` here based on the configuration, but instead directly used hard coding to pass the values ? 

If so (IIUC), The reason for not doing so:

- We want to change the parameter transfer logic uniformly when the default scheduler fully supports balanced scheduling. 
-The current hard coded default values will not break the original logical semantics

Please correct me if i'm wrong. 
Any suggestion is appreciated.

Reply:
IIUC, you plan to modify this parameter in the commit introducing the configuration? Sounds good from my side.

Reply:
Got it, Thank you confirmation~